vllm/model_executor/layers/mamba/linear_attn.py

```diff
@@ -124,9 +125,7 @@ def mamba_type(self) -> str:
         return "linear_attention"
 
     def get_attn_backend(self) -> type["AttentionBackend"]:
-        from vllm.v1.attention.backends.linear_attn import LinearAttentionBackend
-
-        return LinearAttentionBackend
+        return self.mamba_attn_backend
```

Comment:
I notice all the `get_attn_backend` implementation is the same, why note implement it in MambaBase class?